applications/anagolay-project-idiyanale-phase-1.md

```diff
@@ -0,0 +1,368 @@
+|    0e. | Article                        | We will publish an article that explains what was done as part of the grant, with the focus on the developers' community                                                                                                                                                                                                                                                                             |
+|     1. | Workflow: execution            | [See here](#workflow-execution)                                                                                                                                                                                                                                                                                                                                                                      |
+|     2. | Anagolay CLI: Operation Part 2 | [See here](#anagolay-cli-operation-part-2)                                                                                                                                                                                                                                                                                                                                                           |
+|     3. | Demo nodejs app part 3         | Improve Part 2 with automatic Workflow execution which will produce correct Proofs.                                                                                                                                                                                                                                                                                                                  |
+|     4. | Demo web app                   | Simple web app which will use the Workflow automatic execution and produce correct Proofs. Tech used Svelte or React with Typescript and Bulma for CSS. UI mockup is [here](https://bafybeib3sisk6hoejqm4ki46m6jtnzris6psrlrpqcldayorupwuqtcuui.ipfs.dweb.link/Home-page.png) and [here](https://bafybeigbep6y37ghbgmmwxtr5d2jbxi2t7rtyb3m7d3qvxaoxzve73fcry.ipfs.dweb.link/page-image-uploaded.png) |
+
+
+
+> __NOTE__: All the apps, _Nodejs_, and _Web app_ when executed must produce the same Proofs. WHY? Because if they use the same data and the same Workflow they must produce the same output. Same Workflow means using the same Operation manifest which means using the same Operation Version. 100% the same code execution!
+> 
+### Workflow: execution
+Execution of the Workflow manifest created in [#3](#workflow-manifest-generation). We will implement basic recursive and automatic execution of the Workflow only for `SYSTEM` Operations. The execution will load all the dependencies and execute them in the correct order.  We will NOT implement any kind of optimizations like caching or memoization to gain a boost in speed.  
```

Comment:
The link is broken, could you fix it? Otherwise it's hard to follow what is meant.

Reply:
thanks, fixed now.